Examples/Tutorials/Tutorial-PcapFiles/Makefile.windows

```diff
@@ -2,8 +2,8 @@ include ../../../Dist/mk/PcapPlusPlus.mk
 
 # All Target
 all:
-	g++.exe $(PCAPPP_INCLUDES) -c -o main.o main.cpp
-	g++.exe $(PCAPPP_LIBS_DIR) -static-libgcc -static-libstdc++ -o Tutorial-PcapFiles.exe main.o $(PCAPPP_LIBS)
+	$(CXX) $(PCAPPP_BUILD_FLAGS) $(PCAPPP_INCLUDES) -c -o main.o main.cpp
+	$(CXX) $(PCAPPP_LIBS_DIR) -static-libgcc -static-libstdc++ -o Tutorial-PcapFiles.exe main.o $(PCAPPP_LIBS)
```

Comment:
Please put back g++.exe instead of $(CXX) to make it simpler to understand

Reply:
Ther are strong reassons to not to do this:  
1. We have at least 40 publicly available C++ compilers besides g++
2. As i heve mentioned in PR description `CXX` is a **traditionl** name for c++ compiler. My arguments are:  
  * `G++` env var name is not portable across unix systems but `CXX` is (this is my actual motivation as i have had this case on my work): https://stackoverflow.com/questions/2821043/allowed-characters-in-linux-environment-variable-names
*  To enshure You that it is a **traditional** use case i have found next code bases that refers to c++ compiler as `CXX`:
https://github.com/nlohmann/json/blob/develop/Makefile
https://github.com/BVLC/caffe/blob/master/Makefile
https://github.com/grpc/grpc
https://github.com/Microsoft/CNTK/blob/master/Makefile
https://github.com/bitcoin/bitcoin/blob/master/configure.ac  
* In fact this names (`CC and CXX`) are so common that CI (Travis)  is atomatically exports them to test env(please refere to passed tests output):  

**662.9** :
```
50 $ export CXX=clang++
51 $ export CC=clang
```
**662.1** :
```
446 $ export CXX=g++
447 $ export CC=gcc
```

Reply:
@seladb But generally i not strongly against that examples may be written as such but i really like consistency too.

Reply:
I totally understand, that's why I support your idea of replacing `G++` with `CXX` throughout the code. However, in the tutorial examples I intentionally put `g++` for Linux/Mac and `g++.exe` for Windows so it'll be  it easier for people with poor C++ experience to run them.

That's why I asked to put back `g++` and `g++.exe` just for the tutorial examples (and not for all examples)